internal/controllers/phase_reconciler.go

```diff
@@ -361,14 +361,36 @@ func (r *PhaseReconciler) teardownPhaseObject(
 		if !r.ownerStrategy.IsOwner(owner.ClientObject(), currentObj) {
 			return true, nil
 		}
+		// Option 1: Merge patch, only provide the ownerReference to patch
+		object := &unstructured.Unstructured{}
+		object.SetOwnerReferences(currentObj.GetOwnerReferences())
+		r.ownerStrategy.RemoveOwner(owner.ClientObject(), object)
+		objectPatch, err := json.Marshal(object)
+		if err != nil {
+			return false, fmt.Errorf("creating patch: %w", err)
+		}
+		if err = r.writer.Patch(ctx, currentObj, client.RawPatch(
+			types.MergePatchType, objectPatch,
+		)); err != nil {
+			return false, fmt.Errorf("removing external object owner reference: %w", err)
+		}
```

Comment:
This is not an SSA (as requested) for technical reasons, @weherdh outlined in the backing jira issue:

IIUC It is not possible to only remove some known fields from an object using SSA if the full object was created with the same `fieldowner`, as long as the full shape of the object is unknown because every field that is left unspecified in the request, will get removed from the object.

@thetechnick: Would you agree, that using a MergePatch instead is the correct way to go here ?

Reply:
Merge patch is fine for this case, in the end we just want to surgically remove the owner and cache label and nothing else.
But the label is not yet removed here, just the owner.